### PR TITLE
api-server: add the fallback page

### PR DESCRIPTION
**Pull request by cwen0**
Signed-off-by: cwen0 <cwenyin0@gmail.com>

### What problem does this PR solve?
 

* use `index.html` as the fallback page

### CODE REVIEW

File: tools/assets_generate/main.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ import (
 	"net/http"
 	"os"
 
-	"github.com/shurcooL/vfsgen"
+	"github.com/cwen0/vfsgen"
```
Comment (yeya24) [inline tools/assets_generate/main.go:21]: Is this change intended? You'd like to use own fork here?

Comment (cwen0) [reply tools/assets_generate/main.go:21]: The origin repo doesn't support setting the fallback and the repo is rarely updated, but we need this feature. https://github.com/cwen0/vfsgen/commit/c35322782328a576b31b802094d540f346ac0b39

Comment (cwen0) [reply tools/assets_generate/main.go:21]: I will try to push this pr to the origin repo.
